### PR TITLE
DBAL-2626 Add PHPUnit config for ContinuousPHP OracleDB testing

### LINKED ISSUE #2626: [RFC] testing Oracle DB on travis ci
Apparently, someone has a workaround for testing Oracle DB on travis ci as well https://github.com/cbandy/travis-oracle

an Oracle account is required and the Oracle username and password has to be in the build environment variables either as hidden repository settings or encrypted variables

### PR DESCRIPTION
**Pull request by deeky666**
Closes #2626.

### CODE REVIEW

**Comment by deeky666** [comment]
Seems to work. The error was expected and has to be fixed (it is a test error on Oracle 11g).

**Comment by photodude** [comment]
Is there a reason for not using the new convention we set up for the config file naming?
- expected `tests/<service>/<db_type>.<db_version>.<dbal_driver_name>.<service>.xml`
- found `tests/continuousphp/oci8.phpunit.continuousphp.xml`

Does continuousphp not use a yml config file?

**Comment by deeky666** [comment]
@photodude I did not care about convention here because there is only 1 DB version and only 1 driver we want to test here.
Continuousphp does not have a config file, no. It is all configured via their web frontend.

**Comment by deeky666** [comment]
Though I can rename the phpunit config file is that is desired.

**Comment by Ocramius** [comment]
Gotta love OracleDB's expressiveness:

```
1) Doctrine\Tests\DBAL\Functional\Driver\OCI8\OCI8ConnectionTest::testLastInsertIdAcceptsFqn 
Exception: [Doctrine\DBAL\Exception\DriverException] An exception occurred while executing 'CREATE TABLE DBAL2595(id NUMBER DEFAULT dbal2595_seq.NEXTVAL, foo NUMBER)': 
 
ORA-00984: column not allowed here 
 ```

File: tests/continuousphp/oci8.phpunit.continuousphp.xml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:noNamespaceSchemaLocation="http://schema.phpunit.de/4.8/phpunit.xsd"
+         backupGlobals="false"
+         colors="true"
+         bootstrap="../../vendor/autoload.php"
+>
+<php>
```
Comment (Ocramius) [inline tests/continuousphp/oci8.phpunit.continuousphp.xml]: Alignment is a bit borked all over the file ;-)

**Review by Ocramius — CHANGES_REQUESTED** [review]
Need it to be green before we merge it, obviously.

**Comment by photodude** [comment]
Just a consistency item, if that is the convention that we want to try to stick to.

Also, isn't there two drivers to test, `PDO_OCI` and `OCI8`?

**Comment by deeky666** [comment]
@Ocramius I wanted to add a followup PR that fixes the test but I can do it right away with this PR if you want to.
@photodude you are right but the `PDO_OCI` PHP is very buggy since years and even was marked "experimental" until some time ago (no clue why they changed this). With `PDO_OCI` our testsuite is even segfaulting. Maybe we should remove support for it altogether. Also continuousphp does not have the driver installed so we won't be able to test against it out of the box anyways.

File: tests/continuousphp/oci8.phpunit.continuousphp.xml (hunk 1 above)
Comment (deeky666) [inline tests/continuousphp/oci8.phpunit.continuousphp.xml]: Fixed.

**Comment by deeky666** [comment]
@Ocramius fixed the failing test. It's green now :)

**Comment by Ocramius** [comment]
LGTM! Finally some CI for OracleDB as well. I know we'll regret it, but at least we can enjoy a few moments of glory :-)

**Pull request merged**